### ISSUE TITLE
Runtime problem: enum ToString() behaves unexpectedly

### ISSUE THREAD

**Issue by strygo** [bridgedotnet/Bridge#893]
For example:

```
public enum TestEnum
{
    Test,
}

public void Test()
{
    Console.Log(TestEnum.Test.ToString());
}

```

Expected: Test
Observed: 0

It looks like Enum.ToString(Type, Enum) works, however it is clunky. If the above code could be mapped to this alternate method, I would find that preferable. Even if the following case didn't work, I would say it is still better than the current behavior:

```
object o = TestEnum.Test;
string s = o.ToString();
```

**Comment by geoffreymcgill** [comment]
.NET Fiddle of the same code in pure C#.

https://dotnetfiddle.net/bEynkX

We will investigate.

**Comment by geoffreymcgill** [comment]
This may not be as simple as it seems to implement. Adjustments to the Translator logic is required. 

We discussed this issue at our Monday team meeting, and we will investigate some options this week. If there's an easy fix, the change will make it into the 1.11 release, otherwise we'll have to bump to 1.12.
